be2-scala/docs/README.md

```diff
@@ -224,22 +224,117 @@ val f: Future[GraphMessage] = (dbActor ? DbActor.Write(channel, message)).map {
 	)
 }
 
-// Await.result waits for <duration> for the future <f> to complete. It returns the value contained by the future (here `GraphMessage`) if the latter is succesful, or throws if the Future termintes withtout being successful (i.e. either Failure or Timeout) 
+// Await.result waits for <duration> for the future <f> to complete. It returns the value contained by the future (here `GraphMessage`) if the latter is successful, or throws if the Future terminates without being successful (i.e. either Failure or Timeout) 
 Await.result(f, duration)
 ```
 
 
 
 :information_source: the database may easily be reset/purged by either deleting a folder corresponding to a channel or by deleting the `database` folder entirely
 
-## Debugging Tips
+## 4.	Git(Hub)
```

Comment:
I would move this section to the global docs of the project, as it's true for everyone.
I would also consider simply pointing to one of the many Git tutorials.

I'm concerned providing a little bit of information just to get started might delay learning about Git properly, which is arguably what students should be doing at this point :) 
What do you think?

Reply:
Yep! Putting this in the global docuementation file could be a good idea :) 

I only wrote very basic git commands in my opinion, but I think linking a git guide with what we have currently could be a good compromise ^^